server/src/main/java/io/druid/segment/indexing/DataSchema.java

```diff
@@ -58,6 +59,7 @@ public DataSchema(
       @JsonProperty("parser") Map<String, Object> parser,
       @JsonProperty("metricsSpec") AggregatorFactory[] aggregators,
       @JsonProperty("granularitySpec") GranularitySpec granularitySpec,
+      @JsonProperty("rollup") Boolean rollup,
```

Comment:
IMO, TuningConfig doesn't essentially change the data itself, only something like partition, format, performance, etc.
But rollup determines if we should keep the detail of the data like we choose how many dimension from columns, so I think it should be part of the data schema